core/state_transition.go

```diff
@@ -366,6 +371,23 @@ func (st *stateTransition) preCheck() error {
 			}
 		}
 	}
+	// Check that EIP-7702 authorization list signatures are well formed.
+	if msg.AuthList != nil {
+		if msg.To == nil {
+			return fmt.Errorf("%w (sender %v)", ErrSetCodeTxCreate, msg.From)
+		}
+		if len(msg.AuthList) == 0 {
+			return fmt.Errorf("%w (sender %v)", ErrEmptyAuthList, msg.From)
+		}
+		for i, auth := range msg.AuthList {
+			switch {
+			case auth.R.BitLen() > 256:
+				return fmt.Errorf("%w: address %v, authorization %d", ErrAuthSignatureVeryHigh, msg.From.Hex(), i)
+			case auth.S.BitLen() > 256:
+				return fmt.Errorf("%w: address %v, authorization %d", ErrAuthSignatureVeryHigh, msg.From.Hex(), i)
+			}
+		}
```

Comment:
This cannot occur. The `R` and `S` are `uint256.Int`'s, max bitlen is `256` by definition. If this check must exist, then it must exist somewhere earlier in the pipeline

Reply:
Right, I think this is left over from using `big.Int`.